modules/tinymce/src/core/test/ts/browser/dom/SelectionQuirksTest.ts

```diff
@@ -64,8 +65,21 @@ describe('browser.tinymce.core.dom.SelectionQuirksTest', () => {
     TinySelections.setSelection(editor, [ 0, 0, 0 ], 0, [ 0, 0 ], 0);
     TinyContentActions.keyup(editor, Keys.left(), { shift: true });
     assertNormalizeCounter(0);
-    TinyContentActions.keyup(editor, 17, { }); // single ctrl
-    assertNormalizeCounter(1);
+    const isMac = Env.os.isMacOS() || Env.os.isiOS();
+    TinyContentActions.keyup(editor, isMac ? 224 : 17, { }); // single ctrl
+    assertNormalizeCounter(0);
     TinyAssertions.assertSelection(editor, [ 0, 0 ], 0, [ 0, 0 ], 0);
   });
+
+  it('TINY-4550: Normalization should not run after selecting all with keyboard shortcut when there is only an image in the content', () => {
+    const editor = hook.editor();
+    resetNormalizeCounter();
+    editor.setContent('<p><img src="https://www.tiny.cloud/images/glyph-tinymce@2x.png" alt="" width="354" height="116"></p>');
```

Comment:
Maybe use a data url or just `about:blank` since having tests that does a http request out would be a think that would be nice to avoid if the internet is flaky or slow. The `about:blank` is an approach I've used else ware sure the image gets broken since that is not a real url but it doesn't really matter for the tests. Also the image size is irrelevant so it could just be `<p><img src="about:blank"></p>` short and clean.

Reply:
Nice, I like it!
Done!